scripts/oecd/regional_demography/gen_place_mapping_stats.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+## PART 1: JUST THE RESOLVER
+
+df = pd.read_csv("geos_resolved.csv")
+
+print("\nOECD->DCID resolved\n")
```

Comment:
Make it more human readable?

"Count of resolved DCIDs"

Can we group the IDs by namespace (i.e., first split by '/') and count?